pegasus_wrapper/resource_request.py

```diff
@@ -80,6 +80,9 @@ class SlurmResourceRequest(ResourceRequest):
     num_gpus: Optional[int] = attrib(
         validator=optional(in_(Range.at_least(0))), default=None, kw_only=True
     )
+    job_time: Optional[str] = attrib(
+        validator=optional(instance_of(str)), default="1:00:00", kw_only=True
```

Comment:
It's generally best to avoid using strings for things which are not strings. :-). Perhaps make this `job_time_in_minutes: int` and then convert it to the string form in the Slurm-specific code?  Also, I think a higher default is better - say 24 hours.  Otherwise people who don't think to specify this will frequently have their jobs killed.